src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -126,7 +129,7 @@
   (map (fn [{:keys [phone-number-hash whisper-identity address]}]
          (let [contact (contacts-by-hash phone-number-hash)]
            (assoc contact :whisper-identity whisper-identity
-                          :address address)))
+                  :address address)))
```

Comment:
if you also put :whisper-identity association on its own line it will look nicer